ci/travis_release_test.sh

```diff
@@ -29,4 +29,4 @@ sudo apt-get install apt-transport-https
 sudo apt-get update
 sudo apt-get install dotnet-sdk-2.2
 
-dev/release/run-test.rb
+ARROW_TEST_SOURCE_GLIB=yes dev/release/run-test.rb
```

Comment:
I've enabled the test for GLib configure. Because this PR reduces the test time of this option.
But if you don't want to enable this option, I will remove this.

Reply:
It just increase just 1min. We can always run this test now.
Could you remove `omit` from `02-source-test.rb`.?

Reply:
I've removed it.

Reply:
Could you also remove this change?

Reply:
Oh...sorry. I've removed this change.